tasks/token-list.ts

```diff
@@ -0,0 +1,99 @@
+import { TokenList, schema, TokenInfo, Version } from "@uniswap/token-lists";
+import Ajv, { ErrorObject } from "ajv";
+import addFormats from "ajv-formats";
+import { task } from "hardhat/config";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { ERC20 } from "../typechain/ERC20";
+import { EXTERNAL_USD_STABLE, tokenLogoUrl, wBTC_address, wETH_address } from "../utils/Constants";
+import { getBdEu, getBdUs, getBdx, getERC20 } from "../utils/DeployedContractsHelpers";
+
+export function load() {
+  task("print:tokenList").setAction(async (args, hre) => {
```

Comment:
NIT
```suggestion
  task("print:amm:tokenList").setAction(async (args, hre) => {
```

In case we'll add more in the future